velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -35,6 +35,26 @@ class ContainerRowSerdeTest : public testing::Test,
     return position;
   }
 
+  std::vector<HashStringAllocator::Position> serializeWithPositions(
```

Comment:
Is it necessary to add WithPositions to the name of this method? Can it be just 'serialize'?

Reply:
There is another method named serialize with the same input parameters.

Reply:
I see. An existing serialize() writes all rows together and returns a position at the start of this combined write. This one writes each row individually and returns positions for individual rows.

Perhaps, we could remove the existing method and rename this one to 'serialize' + add a boolean indicating whether to serialize all rows together and return a single 'position' or serialize each row individually and returns per-row positions.

Either way, it would be helpful to add comments to clarify how these 2 methods differ from each other.

Not a big deal though.

Reply:
I've added comments to clarify the difference.